packages/backend/src/internal.ts

```diff
@@ -9,12 +9,7 @@ export { debugRequestState } from './tokens/request';
 
 export type { AuthenticateRequestOptions } from './tokens/types';
 
-export type {
-  SignedInAuthObjectOptions,
-  SignedInAuthObject,
-  SignedOutAuthObject,
-  AuthObject,
-} from './tokens/authObjects';
+export type { SignedInAuthObjectOptions, SignedInAuthObject, SignedOutAuthObject } from './tokens/authObjects';
```

Comment:
🔧 This change will break existing application importing from `/internal`. Even though they should not use it, should we keep it and add a deprecation warning to remove it later?
cc: @nikosdouvlis